olm/addDigests.sh

```diff
@@ -124,10 +124,22 @@ yq -riY "( .spec.relatedImages ) += [${RELATED_IMAGES}]" ${CSV_FILE}
 yq -riY "( .spec.install.spec.deployments[0].spec.template.spec.containers[0].env ) += [${RELATED_IMAGES_ENV}]" ${CSV_FILE}
 sed -i "${CSV_FILE}" -r -e "s|tag: |# tag: |"
 echo -e "$(cat ${SCRIPTS_DIR}/license.txt)\n$(cat ${CSV_FILE})" > ${CSV_FILE}
-
 echo "[INFO] CSV updated: ${CSV_FILE}"
 
 if [[ ${OPERATOR_FILE} ]]; then
+  # delete previous `RELATED_IMAGES`
+  envVarLength=$(cat "${OPERATOR_FILE}" | yq -r ".spec.template.spec.containers[0].env | length")
+  i=0
+  while [ "${i}" -lt "${envVarLength}" ]; do
+    envVarName=$(cat "${OPERATOR_FILE}" | yq -r '.spec.template.spec.containers[0].env['${i}'].name')
+    if [[ ${envVarName} =~ plugin_registry_image ]] || [[ ${envVarName} =~ devfile_registry_image ]]; then
+      yq -riY 'del(.spec.template.spec.containers[0].env['${i}'])' ${OPERATOR_FILE}
+      i=$((i-1))
```

Comment:
1. I'd remove this decrement and put the following increment into else block.
2. Shouldn't you change the total number of env vars too?

Reply:
1. I've thought about it but this approach sounds more explicit
2. It does not matter, there are no errors, `yq` returns `null` as a string

Reply:
1. I have opposite opinion, but ok.
2. That's logically wrong.

Reply:
2. It is not a c++ or java. We have to processed all records, other doesn't matter

Reply:
That's a source of errors. If `yq` behavior changes or someone add another line that doesn't tolerate index error... but up to you...